File: Examples/DotNET/SDK/VisualBasic/Text/GetTextItems.vb
Imports Aspose.Cloud

Namespace Text
    Class GetTextItems
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get Text Items from a Word Document
            Dim wordsTextItemsResponse As WordsTextItemsResponse = Common.WordsService.WordsTextItems.ReadDocumentTextItems(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsTextItemsResponse.TextItems.List.Count.ToString() + " text items")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Text/GetTextItems.cs
using System;
using Aspose.Cloud;

namespace CSharp.Text
{
    class GetTextItems
    {
        public static void Run()
        {            
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get Text Items from a Word Document
            WordsTextItemsResponse wordsTextItemsResponse = Common.WordsService.WordsTextItems.ReadDocumentTextItems(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsTextItemsResponse.TextItems.List.Count.ToString() + " text items");            
        }
    }
}
